crates/rspack_plugin_javascript/src/visitors/inject_runtime_helper.rs

```diff
@@ -81,6 +81,10 @@ impl<'a> VisitMut for InjectRuntimeHelper<'a> {
         return;
       }
 
+      match word {
+        "_instanceof" => return,
```

Comment:
Because `ts-es5` test case will failed if we don't add these lines(see next `debug_assert`). the function can't be shaken by `dce` visitor, but it is dead code indeed, if you used `passes = 3` the code will be eliminated by swc. Other than that, the `_instanceof` helper doesn't need to inject_runtime_helper, so it is safe to add this early return.

Reply:
![image](https://user-images.githubusercontent.com/17974631/233819965-57d73d35-ec11-4a03-a935-dfa8ff9de541.png)
you could see the `extendStatics` and `__assign` function is eliminated

Reply:
https://play.swc.rs/?version=1.3.53&code=H4sIAAAAAAAAA%2B1a3W%2FbNhB%2Fz1%2FB6KGRUFWJ3TUD4mZdtw7DHoYO6N4EQ5Bl2lYtU5pIJxVW%2F%2B%2B745coS3YStwM2oEIaR%2BTx7ndf5B1dn9NiEV%2Fc01mVZuufV1u21n9fTMktOTb7%2BTOJp0FUbfnKj2Nvk%2BbMm4bk7zMvii7dH1bOabIp59uC8ssf%2BX12uaJFRWt%2BSfnmMkl5w7JElMmSMlqnoqyjj9y7IYsty0ReMuKrtSGhn6qyFjwkSaJhJDX9a5vXNEkCFLzllHBR55nwJmfvZx9pJqI5XeSM%2FlGXIFE0vuXhJQnlv0vOHqIm8NylxZbeEFFv6dkumJxZCIla5ou0XlIRkrQoAr1mUdb%2BXVoTlm4oyZmcGhZtFiOlkYgPZdsNKD4rtOjQzgD5DTKMcclUDiOs3ZnBY7VR3AZs2drRDxyZNRXbmg0tmCgxCkRy6vKu%2BSTZr4bkg6CVDwtCYMPL4o7iH2gwcCyjn%2FBDrOryPiRr2oCt66URLerGAYFWz9mihDgFbjEQT30knnQopE%2BBBCkj%2BaIhkiwV2Yr4tK7LuqscgtHjkz2djX74O18QX7Kdl4wGRhtfCtELaQEhCRGwyTmNugSRWFHmdzSWvm2Drm9df8EMUu2CQfeg3pi6oLZY5VzakMMLfECoMcH3tSKM3huYvmW55x5XgpECyIDvgkVpVRWNj0KVNMdwMkusUqivtsAeP3xOjRQPXyGPXdubZ3cAiVyKbv6qQOQnIEG%2Bx3AoQ2yZ2iTmDu1O%2F73DaIBEfOJ%2BmjMuUpbRcvFtH9XPk%2FfR1oYPbIAt4ehJG2d%2FnTQUGsCZg9QacJH%2FRQHRcYcDwy%2FoAoxa58uVzXTc4OQAOb8lbFsU5NkzIpoKoX1oNrOygIlb4tko9pBArojVfLRK%2BW9ayDQw6p%2BfH6SRMNzNs2%2BxKOlgPS1LBP9Wb3x5njwQ5YIX%2BYzy6yg5VF%2B4bjidWZs7huiUzJFr1e8I5ahkOSG0IEGSb7vv6VGlDfhQPCiqJ2273TV71tGTfjn7aNZ6rnfJC733eRO3AgPyR22LQBdlJexi9TaD0CW3QKKp3xCPy788cmMYAfljdraBoP1PBN3DLGwSWx7DISTD56hXD%2B8HUGJRNv8gUpFnWABbw3QmRv48JDPDeX%2BN1oRTAWqIEt3zfoHdr3Mk%2BS2oJKmQLIFQjHWwh%2BRtXadNgFFgFXFF4jOP7EoQOjNNCsg5uMQkbYUZOwvwtNZgK4MUz9b398w4IMogD%2F1ZSKogIPO4wh5%2FBh%2B24HTM2rGDkg0BqfdZ6E14vmSuSc3YyPrIodK49LujlCXyxZBqECo5LB%2FBfuP2L1FB2VKsJjD5mjD4eP486NbwnXYnzqd77YhrOf54y3FlOaEsx63lVNTtxaUYsqpRVzdMtj1TQJWFHdu0nR9Svq2X4IZy3gQ266DBpfXM9QO%2B%2B2wgT2wc3e03PJqAY5cT91ohFvaG7jojU7d56SgtXQhcXXxSSlm5GND%2BCzArtk6yFf0THPAL9t%2B%2BZ3swknM4RWqazhuITZptRc6WkefIvl%2FlBfWXmGbw%2B5ZchaSs4qupHMCsugoC2AGD7jWCBaDirJHUAl70WjKG%2FbmJPWUkbwpbtJqRo6rdk7dhvlzlUMqEFypyGpB8FcDaJsLGT06dywWGQIIdTYPAXie4UeQibZRuwKREy%2FY9ZpH3PSfUHUjXf10R%2FD4X2cqXTPayCp8shaPi6mZ4fNQfl1KlOSeDczNw6bo%2FJdl9N8wuiYp0Rovnz4c5asv1oZtHH1bS3n0TmQfdACdPCl3IIM3uAOhXJ4FuVMSNpsPTBzxtnqvBmQO8oAqB3DEXYT383w%2FjlwiSqKx4VJWVHwzzTiLIrqMUh6VDsZBuCzEsHgNfJYwSERKVPOokID%2BQK5ltsR15QUYqA1Uky3rrGvO0fR0HQ1c%2FVhVMs2EdjuuBz%2B6gEq34lxLdudCgRlPQQuj9Sr2%2FhveX0%2BMoVVgdjx58DuTZY8FeIyojDHGhdR8DSxxFdWxnOBm12Mc6fjTW8RGsOvjxCw84v742ZgnyX8uvrmCZy1hIqLPH1haJe5LbC%2FJ92x3Yi677W%2BnRc6bp5teOQKsAaJquMHmJvUfZKRzMYffKFBB7eTB4GrQngDzIVbbdkLsyn8PR2iFVx4BsbCyAXpGDe0UrQYbTjcuIQ3Xn9GX46u%2BbVUWB1GVkdOknjykLOhO7VhKGB7Ye7QhElNuLLKDGgc0zJMtOYbrsaIA1yo0qJa%2BClpUud%2FTMyJ3RRY%2BeGgdGXLcvxp3EM2bwVKVmrgFzoQq9qVMtDvV7GK3aAYHUYme6zQwST1Dwp1NBq7GfcjaHctEvQ7IJyRp%2Bxu4l53oskbW38QRHyLp18JzyrO1joCd1eoN3MFfnFVbpyFznkDyw5DKsDT1Y4mG3IUfekPNN1HbREHk4HN3XucD2F1fIAcjlRb7cyp4YOzYF4tGXLtYM8NYLN23NTbzuNTE6sId7%2BBKNF0os2K4Qpzk%2FybplvB6jyxWQ3eSYM6EJVxZ7p0oFlGi7mYNwPV1YeI%2B7r9J7w50Osb6OVmIZW9aowZ2EL29sLiHe6KdI%2FE9uYwTl4kuvYZBH7z62%2FxXm1%2FlOY%2Fg%2FDbS3wF9b3v7XBe7FISpuDWJvGmHwcI%2Fv3jq6q%2BUbIh7QL%2FEfuB7v0CqhdkXCBSTTgS2gf5SPD53b9rvJYPIUdZWSMjfgX4tqOPzVDZOZ0lEqr8WGPHnRJtsFiABp02DyD4Az%2BULYIgAA&config=H4sIAAAAAAAAA32Vy47bMAxF9%2FMVgddd9IF20W2B2RXoHwiyRDtK9TBEKhNjkH8vLduZzJTKzubhFa0rin59Ohy6E5ru5%2BGVH%2Fll0hkh3945gnMkfeFIByZoNNlN1H3a6QkXNGiPUEPXlXSk8whUVfj185fvm6LzKSHsii0WXHTDfF%2FTpDBlQLyLcZSXLAEi4Xv9xnJ6kUCfkgcdHyGlUblIMEKW0kzyXk8I6qwbPLBrDpNYZKGFwKopp0lOiNaRS5HjAragrTLJgsRcBkPuDKKQ67EwIm%2ByhS30ZRzreX%2FkcNa%2BaJKkcKmHw58swGNySGooohkrbDmx0obLblAZqGSp5im52NL9BWAbvEaMOohG1YyBm6sNH0pdHLiBaRY4d7u40wgjW6ucGyR%2FF38gkxNLZrDFwGKweYAbZqCzoGAYuGtE%2FOLIHMWyNE%2BQBonwSWtxGytQzZu58eWCPMDPvFGSW23LCJqObYpz6JN4s3Y50DHZBwl8HJTaOPPcuExtXqIFbhCwYkrBCiiXDzOBbwIl5esM%2Fb9B%2BKLwkmr0qZfHxsQNXw%2Fy2xa73kZ20HF8GwnvpjaLSpgWu%2B9Hb4dAf0rvnfm13KNnB776tXz0Kn%2FaluhCsqWuvf1OlrZZfwI%2Furekfd7fPrpz%2BHtX1lWv%2FwAxEJRcmgYAAA%3D%3D